### ISSUE TITLE
Don't save OAuth secrets to a credentials.txt file

### ISSUE THREAD

**Issue by seanh** [hypothesis/lti#28]
After creating a developer key for us and adding our Canvas app to a course, a school teacher or administrator has to submit a [web form](http://lti.hypothesislabs.com/lti_credentials) giving us a shared secret from Canvas. The secrets submitted via this form are saved to a `credentials.txt` file on the server. A Hypothesis admin then needs to `ssh` into the server, copy the secret and other fields, reformat them, and paste them into the `canvas-auth.json` file (also on the server's local filesystem).

There's a separate issue for the `canvas-auth.json` file. This issue is for the `credentials.txt` file: the `.lti_credentials` form needs to save credentials directly to a Postgres database and not require manual editing by a Hypothesis admin after credentials are submitted, not write them to a text file.

Before we can fix this the app will need to actually have a Postgres db, of course.
